apps/client/src/app/services/data.service.ts

```diff
@@ -55,7 +55,7 @@ import { map } from 'rxjs/operators';
   providedIn: 'root'
 })
 export class DataService {
-  public constructor(private http: HttpClient) {}
+  public constructor(private http: HttpClient) { }
```

Comment:
❓

Reply:
Sorry that was probably prettier I have configured to auto format when saving